src/patterns/dates/index.md.njk

```diff
@@ -82,9 +82,11 @@ To check that the dates users give you are valid, make sure that:
 - past dates are in the past
 - future dates are in the future
 - the first date in a date range is before the second
+- a date with or without a leading zero is accepted
 
 ### How to write dates
 
+Do not use a leading zero in example dates. If you provide an example day number, use a number greater than 12. If you provide an example month number, use a single digit. This will emphasise the distinction between the day field and the month field which can be a problem for some users, particularly if accustomed to other date formats. It will also highlight that leading zeros aren't required.
 See the [GOV.UK style for writing dates](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates) and [date ranges]( https://www.gov.uk/guidance/content-design/writing-for-gov-uk#date-ranges).
 
```

Comment:
```suggestion

If you give an example date, use 13 or more for the day and 9 or less for the month - for example '27 3 2007'. This helps users enter the date in the correct order and without leading zeroes.

```

Reply:
I think there's a subtle difference which is that we're not trying to get them to leave out leading zeroes, since we've explicitly said accept both, but it's showing them they don't need to have leading zeroes.

So I'd say:

"... This helps users enter the date in the correct order and shows them they do not need to include leading zeroes."